docs/en/stack/ml/index-custom-title-page.html

```diff
@@ -0,0 +1,218 @@
+    }
+
+    #guide h3.gtk {
+    margin-top: 0;
+  }
+
+  .mb-4, .my-4 {
+    margin-bottom: 0!important;
+  }
+  </style>
+  
+  <div class="legalnotice"></div>
+  
+  <div class="row my-4">
+    <div class="col-md-6 col-12">
+      <p></p>
+      <h2>Analyze your data with Elastic machine learning</h2>
+      <p>
+        Find anomalies and outliers, forecast based on trends, analyze natural language data, and identify areas of interest in your data with machine learning features.
+      </p>
+      <p>
+        <a href="https://www.elastic.co/guide/en/machine-learning/current/machine-learning-intro.html">
+          <button class="btn btn-primary">Get started</button>
+        </a>
+      </p>
+      <p>
+        <a class="inline-block mr-3" href="setup.html">Setup and security</a>
+    </p>
+    </div>
+    <div class="col-md-6 col-12">
+      <img class="w-100" src="https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltdd4cccf507615a8b/63e62da759036b11199d0f6b/ML_cover.jpg" />
+    </div>
+  </div>
+  
+  <h3 class="gtk">Get to know Elastic machine learning</h3>
+  
+  <div class="my-5">
+    <div class="d-flex align-items-center mb-3">
+      <h4 class="mt-3">
+        <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltf6ecdd8b7c1b32c4/63e62879520e5b22386b885f/64x64_Color_icon-machine-learning-64-color.png');"></span>
+        Find anomalies
+      </h4>
+    </div>
+    <ul class="ul-col-md-2 ul-col-1">
+      <li>
+        <a href="ml-getting-started.html">Getting started with anomaly detection</a>
+      </li>
+      <li>
+        <a href="ml-anomaly-detection-job-types.html">Select your analysis type</a>
+      </li>
+      <li>
+        <a href="ml-ad-plan.html">Plan your analysis</a>
+      </li>
+      <li>
+        <a href="ml-ad-run-jobs.html">Run an anomaly detection job</a>
+      </li>
+      <li>
+        <a href="ml-ad-view-results.html">Interpret the results</a>
+      </li>
+      <li>
+        <a href="ml-ad-forecast.html">Forecast future behavior</a>
+      </li>
+    </ul>
+  </div>
+  
+  <div class="my-5">
+    <div class="d-flex align-items-center mb-3">
+      <h4 class="mt-3">
+        <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt566c0864ac509255/63e628ca45406c110ebcc3d0/64x64_Color_icon-automation-with-AI-color-64.svg');"></span>
+        Perform data frame analytics
+      </h4>
+    </div>
+    <ul class="ul-col-md-2 ul-col-1">
+      <li>
+        <a href="ml-dfa-finding-outliers.html">Find outliers</a>
+      </li>
+      <li>
+        <a href="ml-dfa-regression.html">Predict numerical values with regression</a>
+      </li>
+      <li>
+        <a href="ml-dfa-classification.html">Predict classes with classifiation</a>
+      </li>
+      <li>
+        <a href="ml-trained-models.html">Manage trained models</a>
```

Comment:
Would `Trained models` be better here - that would be consistent with the title of the page it is linking too? The first I saw this I was expecting it to be a link to the page under Model management in the UI (which I associate more with NLP trained models rather than ones from data frame analytics jobs).

Reply:
Fixed in [bf95934](https://github.com/elastic/stack-docs/pull/2326/commits/bf95934e55424a66e6cdfc8ffb10cdb303d5fd6d)